src/main/kotlin/net/badgersmc/trivia/infrastructure/bukkit/ChatListener.kt

```
@@ -22,17 +22,14 @@ class ChatListener(
     private val chatPlatform: ChatPlatform,
 ) : Listener {
 
-    private val plain = PlainTextComponentSerializer.plainText()
-
-    @EventHandler(priority = EventPriority.LOWEST, ignoreCancelled = true)
-    fun onChat(event: AsyncChatEvent) {
+    @EventHandler(priority = EventPriority.LOWEST, ignoreCancelled = false)
+    fun onChat(event: AsyncPlayerChatEvent) {
```

detekt (1.23.8)

[warning] The function onChat appears to be too complex based on Cyclomatic Complexity (complexity: 26). Defined complexity threshold for methods is set to '25'

(detekt.complexity.CyclomaticComplexMethod)